update_prevalence.py

```diff
@@ -1328,11 +1334,11 @@ def get_partially_vaccinated(total_shots, total_fully_vaccinated, shots_for_full
         # add population data
         place = data.get_canada_region_place(region)
         if place.population != 0:
-            raise ValueError(f"Duplicate population info for {place!r}: {region.Population}")
+            raise ValueError(f"Duplicate population info for {place!r}: {region.pop}")
```

Comment:
This looks like it was wrong beforehand, but would only blow up when we're already handling errors.